Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrDriver.kt

```diff
@@ -36,12 +37,21 @@ class StubIrContext(
                     }
     ).precompileHeaders()
 
+    // TODO: Used only for JVM.
```

Comment:
Why it is a TODO?

Reply:
Because it has nothing to do (no pun intended) with the purpose of this PR. I'll do the cleanup after the feature is completed.